### ISSUE TITLE
[Feature Request] Auto reproduce audio messages

### ISSUE THREAD

**Issue by gabriele-v** [slowlydev/f1-dash#87]
It would be useful to auto reproduce & play audio messages immediately once they are received.

Thanks

**Comment by slowlydev** [comment]
Hey there, thanks for the issue!
I also had this idea but thanks for the suggestion.
A thing I would add to this feature that u need to/can select the drivers of which u want team radios to auto play.

Feel free to implement this yourself, but I would suggest doing that of of the v2 branch.

If not, I will do this some time after v2 release.

**Comment by gabriele-v** [comment]
Unfortunately I don't have typescript experience, only C++ and C#, but I'll give it a try if I have some spare time.

**Comment by slowlydev** [comment]
No worries :)

**Comment by aperonam** [comment]
Hey @slowlydev did you start  working on this? I would be happy to take this and implement it

**Comment by slowlydev** [comment]
@aperonam hey there!

No I did not start working on this, but I have some ideas for it. 

As already mentioned I think there should be a toggle for it as well as an option to select which drivers should auto play. 

And so it can be configured on the settings page and saved to localStorage.

Feel free to pick this issue up :)

**Comment by aperonam** [comment]
Yep I was thinking something like that. But instead of doing it in settings I was thinking on directly adding a mute / volume icon to the driver's number-name badge to toggle this, what do you think? All drivers would be "muted" by default.

**Comment by slowlydev** [comment]
Okay yeah definitely also should integrate it somehow directly on the dashboard itself. 

Because I want to add extensive filtering of all things (map, audio, race control, leaderboard) I want to add "headers" to all the sections but I have not comeuppance with a food design yet. This is a small design thingy I just did. The filter icons should only show hen hovering in that area (team audios and the popup should show when clicking on the filter icon)

<img width="501" alt="image" src="https://github.com/slowlydev/f1-dash/assets/61624214/6e73dd02-6f41-44c2-bd1a-fa48d00d388a">

But I am looking forward to see your approach and then we can make adjustments later on :)

**Comment by cristianobattini** [comment]
Hi, as a F1 fan who is also a dev I would be pleased to introduce this feature. I was thinking about a toast that displays whenever a new team radio is retrieved and play automatically the radio message. I have still to figure out where to position it in the mobile view or how can I implement in other ways, but  I will find out a method. I'll keep you posted!
